src/com/javabasic/lesson10/homework/Task03.java

```diff
@@ -0,0 +1,25 @@
+package com.javabasic.lesson10.homework;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Task03 {
+    public static final BufferedReader READER = new BufferedReader(
+            new InputStreamReader(System.in)
+    );
+
+    public static void main(String[] args) throws IOException {
+        System.out.print("Введите число от 0 до 42 : ");
```

Comment:
if you ask a number in range, validate the input, what if number will be more than 42?